public/styles/components/_pagination.scss

```diff
@@ -0,0 +1,62 @@
+.bsds-pagination {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  li {
+    &.bsds-pagination-page,
+    &.bsds-pagination-stepper {
+      margin: 0;
+    }
+
+    &.bsds-pagination-page {
+      [class*='bsds-button'],
+      [class*='bsds-link'] {
```

Comment:
thoughts on setting display here so pagination page links are the same height as pagination page buttons?